src/improvements/template/UnPauseSuperchainConfig.sol

```diff
@@ -1,64 +0,0 @@
-// SPDX-License-Identifier: MIT
-pragma solidity 0.8.15;
-
-import {VmSafe} from "forge-std/Vm.sol";
-import {stdToml} from "forge-std/StdToml.sol";
-import {ISuperchainConfig} from "lib/optimism/packages/contracts-bedrock/interfaces/L1/ISuperchainConfig.sol";
-import {
-    IDeputyGuardianModule,
-    IOptimismPortal2
-} from "lib/optimism/packages/contracts-bedrock/interfaces/safe/IDeputyGuardianModule.sol";
-
-import {L2TaskBase} from "src/improvements/tasks/types/L2TaskBase.sol";
-import {SuperchainAddressRegistry} from "src/improvements/SuperchainAddressRegistry.sol";
-import {Action} from "src/libraries/MultisigTypes.sol";
-
-/// @title UnPauseSuperchainConfig
-contract UnPauseSuperchainConfig is L2TaskBase {
-    using stdToml for string;
-
-    // /// @notice Mapping of chain ID to configuration for the task.
-    //
-    /// @notice Returns the string identifier for the safe executing this transaction.
-    function safeAddressString() public pure override returns (string memory) {
-        return "FoundationOperationsSafe";
-    }
-
-    /// @notice Returns string identifiers for addresses that are expected to have their storage written to.
-    function _taskStorageWrites() internal pure override returns (string[] memory) {
-        string[] memory storageWrites = new string[](2);
-        storageWrites[0] = "SuperchainConfig";
-        storageWrites[1] = safeAddressString();
-        return storageWrites;
-    }
-
-    /// @notice Sets up the template with implementation configurations from a TOML file.
-    function _templateSetup(string memory taskConfigFilePath, address rootSafe) internal override {
-        super._templateSetup(taskConfigFilePath, rootSafe);
-    }
-
-    /// @notice Write the calls that you want to execute for the task.
-    function _build(address) internal override {
-        // Load the DeputyGuardianModule contract.
-        IDeputyGuardianModule dgm = IDeputyGuardianModule(superchainAddrRegistry.get("DeputyGuardianModule"));
-        dgm.unpause(); // Unpause the SuperchainConfig contract through the DeputyGuardianModule.
-    }
-
-    /// @notice This method performs all validations and assertions that verify the calls executed as expected.
-    function _validate(VmSafe.AccountAccess[] memory, Action[] memory, address) internal view override {
```

Comment:
@blmalone this template is using old code I don't even know this compiling in the past?

Reply:
Oh you are right, I was likely on the incorrect file. My bad for this you can ignore this comment haha starting to be tired!